### ISSUE TITLE
Is there a way to control to property `aria-describedby` for the dive with `textbox` as role?

### ISSUE THREAD

**Issue by alexxxmf** [sstur/react-rte#352]
I'm trying to fix some accessibility issues but maybe I'm missing something doesn't seem the component provides a way to control the aria attribute assigned to the `textbox`. At the moment I'm getting something like this `placeholder-{randomId}`

**Comment by alexxxmf** [comment]
<img width="472" alt="Screenshot 2019-11-29 at 16 16 07" src="https://user-images.githubusercontent.com/11931541/69881035-97a02900-12c3-11ea-992c-c9fbb89e81ca.png">
